### ISSUE TITLE
The Legend of Zelda: Ocarina of Time

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#377]
Originally reported on Google Code with ID 378

```
Describe your system:
 - Operating System (be specific): Ubuntu 10.04
 - Machine type (32-bit or 64-bit):32-bit
 - Mupen64Plus version:1.99.3
 - Plugins used:Rice

For visual artifacts, give:
- game name and CRC/MD5 hash (printed on console)
- description of the artifact (missing polygons, flashing, wrong colors,
etc)

Describe the problem:
After press pause in The Legend of Zelda: Ocarina of Time shows black srceen about
5sec and then i can see pause menu.

And this menu have gliches (screen in attach)
```

Reported by `Virusmater` on 2010-07-24 09:10:26

<hr>
- _Attachment: Снимок-Mupen64Plus OpenGL Video Plugin by Rice v1.99.3.png<br>![Снимок-Mupen64Plus OpenGL Video Plugin by Rice v1.99.3.png](https://storage.googleapis.com/google-code-attachments/mupen64plus/issue-378/comment-0/Снимок-Mupen64Plus OpenGL Video Plugin by Rice v1.99.3.png)_

**Comment by richard42** [comment]
```
This is like Issue 377.
```

Reported by `dorian.fevrier` on 2010-07-24 10:27:04

**Comment by richard42** [comment]
Reported by `dorian.fevrier` on 2010-08-20 07:06:48
- **Status changed**: `Duplicate`
- **Merged into**: #377
